tsconfig.json

```diff
@@ -22,6 +22,6 @@
       "@/*": ["./src/*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts", "src/pages/home.tsx"],
```

Comment:
why do you need to specify this file here?

we have this patter which should indexate the file -- "**/*.tsx"